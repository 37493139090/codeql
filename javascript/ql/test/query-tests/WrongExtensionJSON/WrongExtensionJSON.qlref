query: LanguageFeatures/WrongExtensionJSON.ql
postprocess: utils/test/InlineExpectationsTestQuery.ql
