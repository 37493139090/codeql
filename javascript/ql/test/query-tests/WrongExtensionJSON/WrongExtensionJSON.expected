#select
| tst1.js:1:1:3:1 | {\\n  "x": 42\\n} | JSON data in file with extension 'js'. |
testFailures
| tst1.js:1:1:3:1 | JSON data in file with extension 'js'. | Unexpected result: Alert |
